VMEncryption/main/patch/UbuntuPatching.py

```diff
@@ -89,8 +100,14 @@ def install_extras(self):
                     'procps',
                     'psmisc']
 
-        cmd = " ".join(['apt-get', 'install', '-y'] + packages)
-        self.command_executor.Execute(cmd)
+        if self.packages_installed(packages):
```

Comment:
This can prevent packages to get upgraded. It can become an issue if image package has some bug and we need the new package that contains the fix.
Should not happen often but something to note.

Reply:
Yes, this shifts any package update responsibility to the image builder and system administrator who can do it outside of the time constraints imposed by the guest agent, or outside of network connectivity constraints that break this today.